src/simulators/matrix_product_state/svd.cpp

```diff
@@ -140,9 +141,28 @@ double reduce_zeros(cmatrix_t &U, rvector_t &S, cmatrix_t &V,
   return discarded_value;
 }
 
+void validate_SVdD_result(const cmatrix_t &A, const cmatrix_t &U,
```

Comment:
Added this function to avoid applying AER::Utils::dagger to V all the time in lapack_csvd_wrapper(